__tests__/auth.js

```diff
@@ -31,7 +34,22 @@ describe('Authentication', () => {
     response.resetMocked();
   });
 
-  beforeAll(() => osjs().then(c => (core = c)));
+  beforeAll(() =>
```

Comment:
I now see that I don't have linting for the test folders... and no editorconfig :)

This should be two spaces of indentation.